### ISSUE TITLE
Consistent planet icon

### ISSUE THREAD

**Issue by YPetremann** [nicholasgower/planet-muluna#12]
**Is your feature request related to a problem? Please describe.**
Almost every planet (including mods) add some sort of "realist" icon of planets.
With "visible planet" mod, this is even more mandatory to have a "realist" icon

**Describe the solution you'd like**
Changing the icon to be realist enough with some shading to be seen with "visible planet" mod

**Comment by nicholasgower** [comment]
I agree that this is an issue with the current icon. I'm not very experienced with making art assets yet. Once the mod is more playable, I plan to modify my current icon's shading to make it more visible. I will also accept offers from others to make their own icon based on our own moon.

**Comment by nicholasgower** [comment]
I have now added shading to Muluna's icon.
